Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "StipplePlotly"
 uuid = "ec984513-233d-481d-95b0-a3b58b97af2b"
 authors = ["Adrian <e@essenciary.com> and contributors"]
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
I think we should bump to v2, given that it's breaking

Reply:
Ok, I will do so.